### DISCUSSION TITLE
关于捐助模式

### DISCUSSION THREAD

**Discussion by rainpenber** [the1812/Bilibili-Evolved#2543] [综合讨论 / General]
非常看好这个项目，有意向长期支持开发团队进行，请问团队是否有兴趣在爱发电等平台上开启付费订阅，以便更好的进行开发

**Comment by the1812** [comment]
可以考虑开一个, 不过也是纯支持的那种, 订阅了也不会有多什么特权(

**Reply by the1812** [reply]
已支持爱发电: https://afdian.net/@the1812
